scripts/xnat/check_valid_sessions.py

```diff
@@ -13,19 +13,43 @@
 
 Example
 =======
-- Update the cache and generate the baseline report
-  ./check_valid_sessions --update --baseline
+- When running for the first time run 
+  ./check_valid_sessions.py --update 
+  so that the cach (located at experimentsdir) is created 
+
+- Update the cache (stored in experimentsdir) and generate the baseline report
+  ./check_valid_sessions.py --update --baseline
 
 - Use the existing cache to extract 10 in the followup window
- ./check_valid_sessions --num_extract 10 --min 180 --max 540
+ ./check_valid_sessions.py --num_extract 10 --min 180 --max 540
+
+If you have problems running it log in as nicholsn and run 
+module load miniconda
+source activate ncanda-1.0.0
 """
 __author__ = "Nolan Nichols <http://orcid.org/0000-0003-1099-3328>"
 __modified__ = "2015-08-26"
 
 import os
 
+import re
+
+def cmpVersion(version1, version2):
+    def normalize(v):
+        return [int(x) for x in re.sub(r'(\.0+)*$','', v).split(".")]
+    return cmp(normalize(version1), normalize(version2))
+
 import pandas as pd
 
+# might also work for pandas version less than that . I just know it does for this one 
+# when run with update 
+minPandasVersion = "0.13.1"
+# but then crashes later - so then you have to run it with version 
+# minPandasVersion = "0.17.1"
+if cmpVersion(pd.__version__,minPandasVersion) < 0 : 
+    print "ERROR: pandas version too small (Current: %s; Required: %s)!" % (pd.__version__,minPandasVersion)
+    exit(1) 
```

Comment:
please remove this debugging code as we identified the issue was not due to the pandas version